src/zeek.bif

```diff
@@ -315,6 +315,7 @@ zeek::RecordValPtr zeek::detail::build_dummy_conn_record()
 	id_val->Assign(1, zeek::val_mgr->Port(ntohs(0), TRANSPORT_UDP));
 	id_val->Assign(2, zeek::make_intrusive<zeek::AddrVal>((unsigned int) 0));
 	id_val->Assign(3, zeek::val_mgr->Port(ntohs(0), TRANSPORT_UDP));
+    id_val->Assign(4, 255);
```

Comment:
Indentation buglet. Also, since this `&defaults` to 65535, do we need this?

(I wonder why once upon a time we chose `TRANSPORT_UDP` in the above, not `TRANSPORT_UNKNOWN`, odd.)

Reply:
> (I wonder why once upon a time we chose `TRANSPORT_UDP` in the above, not `TRANSPORT_UNKNOWN`, odd.)

I hadn't noticed that. I wonder if this should be set to `IPPROTO_UDP` then?

Reply:
I wondered too. But the only use for this is as dummy return value when `lookup_connection()` fails. I don't think that's supposed to look like a UDP flow. Most existing uses of `lookup_connection()` don't actually anticipate that it can fail. So all in all I'd still lean toward leaving it out.